src/@batch-flask/ui/duration-picker/duration-picker.component.ts

```diff
@@ -183,12 +191,11 @@ export class DurationPickerComponent implements FormFieldControl<any>,
     }
 
     public validate() {
-        if (this.invalidTimeNumber || this.invalidCustomDuration) {
+        if (this.invalidTimeNumber || this.invalidCustomDuration || (this.required && !this.time)) {
             return {
                 duration: "Invalid",
             };
         }
-        return false;
```

Comment:
what does it return now?

Reply:
Null (https://angular.io/api/forms/Validator#validate)

Reply:
False does the same thing but not returning meets the standard better.

Reply:
well acutally not returning means it return `undefined` you should try to return null explicitly(And may add the return type defined in the doc to that function)

Reply:
Ah yea